core/prelude/types/int.carbon

```diff
@@ -32,6 +32,11 @@ impl forall [N:! IntLiteral()] Int(N) as As(IntLiteral()) {
   fn Convert[self: Self]() -> IntLiteral() = "int.convert_checked";
 }
 
+// TODO: Allow as an implicit conversion if N > M.
+impl forall [M:! IntLiteral(), N:! IntLiteral()] Int(M) as As(Int(N)) {
+  fn Convert[self: Self]() -> Int(N) = "int.convert";
+}
```

Comment:
For all of these, and maybe better in a follow-up, would it make more sense to write these as "from N bits to M bits", and make that consistent? I find it a bit surprising which is `N` and which is `M` here.

Or maybe the bit widths should be `From` and `To`? Or `FromBitwidth` and `ToBitwidth`?